CMakeLists.txt

```diff
@@ -175,6 +175,11 @@ if(WIN32)
     )
 endif(WIN32)
 
+if(UNIX AND NOT APPLE)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(LIBSECRET REQUIRED libsecret-1)
```

Comment:
Can we make this optional and fallback to file-based storage if libsecret isn't installed? I suspect it'll be available on almost every modern *nix, but it's nice to not have additional hard dependencies